pmm-app/src/pmm-qan/panel/components/ManageColumns/ManageColumns.tsx

```diff
@@ -102,7 +106,7 @@ export const ManageColumns = (props) => {
   );
 
   return (
-    <div className={!onlyAdd ? 'manage-columns' : 'add-columns'} onClick={(e) => e.stopPropagation()}>
+    <div className={!onlyAdd ? styles.manageColumns : styles.addColumns} onClick={(e) => e.stopPropagation()}>
```

Comment:
Why do we need this `stopPropagation`?

Reply:
We need it here to not table itself, I'll try to think if it could be implemented simplier